article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -361,14 +362,22 @@ object DotcomponentsDataModel {
       relevantBlocks.filter(block => ids(block.id))
     }
 
-    def findPillar(pillar: Pillar, tags: List[Tag]): String = {
-      val isPaidContent = tags.exists(tag => tag.`type` == "Tone" && tag.id == "tone/advertisement-features")
+    def isPaidContent(tags: List[Tag]): Boolean = tags.exists(tag => tag.`type` == "Tone" && tag.id == "tone/advertisement-features")
 
-      if (isPaidContent) "labs"
+    def findPillar(pillar: Pillar, tags: List[Tag]): String = {
+      if (isPaidContent(tags)) "labs"
       else if (pillar.toString.toLowerCase == "arts") "culture"
       else pillar.toString.toLowerCase()
     }
 
+    def findDesignType(designType: Option[DesignType], tags: List[Tag]): String = {
+      // TODO Remove this when changes can be moved to https://github.com/guardian/content-api-scala-client/blob/master/client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala
+      if (isPaidContent(tags)) "AdvertisementFeature"
+      else designType.map(_.toString).getOrElse("Article")
+    }
+
+    article.metadata.designType.map(_.toString).getOrElse("Article")
```

Comment:
Is this line used now?

Reply:
Good spot, that as my reference line I put the temporarily  😨